packages/flutter/lib/src/gestures/mouse_cursor.dart

```diff
@@ -0,0 +1,332 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show hashValues;
+
+import 'package:flutter/foundation.dart';
+
+/// Internal identifiers for the system cursors supported by Flutter.
+///
+/// Each value of [MouseCursorSystemShape] corresponds to a [MouseCursor] object
+/// in [SystemMouseCursors].
+///
+/// This enum is only used for [MouseCursorPlatformDelegate] to implement system
+/// cursors on platforms. To assign widgets with system cursors, use the objects
+/// defined in [SystemMouseCursors] instead.
+///
+/// See also:
+///
+///  * [SystemMouseCursors], which contains usable [MouseCursor] objects that
+///    correspond to values of this type.
+///  * [MouseCursorPlatformDelegate], which uses this type to define how
+///    system cursors are implemented on platforms.
+enum MouseCursorSystemShape {
+  /// The shape that corresponds to [SystemCursors.none].
```

Comment:
is "SystemCursors" supposed to be "SystemMouseCursors"? I couldn't find a definition of SystemCursors.